Telerik.Sitefinity.Amazon/BlobStorage/AmazonBlobStorageProvider.cs

```diff
@@ -26,25 +26,42 @@ public class AmazonBlobStorageProvider : CloudBlobStorageProvider
         /// <param name="config">The collection of parameters (each by its name and value) of the current provider's configuration settings.</param>
         protected override void InitializeStorage(NameValueCollection config)
         {
+            var useIamInstanceRoleValue = "false";
```

Comment:
Optional configuration item to allow the use of the EC2 Instance Profile Role, defaults to false, maintaining backwards compatibility